packages/patternlab/styleguide/source/assets/js/modules/mapLeaflet.js

```diff
@@ -108,15 +122,78 @@ export default (function (window,document) {
           interactive: !isStatic
         })
       .addTo(mymap)
-      .bindPopup(compiledTemplate(infoWindow));
-
-      if (!isStatic) {
-        // popup close onclick is not recognized as a click event inside of the map in order to regain focus
-        // Fire the map focus callback on popup close
-        mymarker.on('popupclose', function() {
-          mymap.fire('focus');
-        });
-      }
+      .bindPopup(compiledTemplate(infoData));
+
+      $(mymarker._icon).on('keydown', function(e){
+
+        if (e.code == 'Tab') {

```

Comment:
```suggestion
        if (e.key == 'Tab' ) {

```
IE doesn't support e.code. I think we can use .key
or add a fallback like (e.code == 'Tab' || e.keyCode == 9 )?